src/App.js

```diff
@@ -9,7 +9,10 @@ const App = () => {
   return (
     <div className="inventory">
       <NotificationsPortal />
-      <RBACProvider appName="inventory" checkResourceDefinitions>
+      <RBACProvider
+        appName={null /* fetch permissions from all scopes */}
```

Comment:
What if we extend the `RBACProvider` to support something like `scopes`, with `*` being all?

Reply:
I was an author of the recent RBACProvider changes actually, and yeah, this is the good suggestion. There were some suggestions from Karel to keep it simple for now though. So there are three values accepted ATM: null for fetching all scopes, undefined for none, and string that represents the name of app and also scope.